content/docs/connectors/keystone.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Authentication Through OpenStack Keystone"
+linkTitle: "OpenStack Keystone"
+description: ""
+date: 2020-09-30
+draft: false
+toc: true
+weight: 2100
+---
+
+## Overview
+
+[Keystone](https://docs.openstack.org/keystone/latest/) is an OpenStack service that provides API client authentication, service discovery, and distributed multi-tenant authorization. 
+
+OpenStack Keystone connector supports `offline_access` and `groups` scopes. To use this connector, you need to create a domain and user with an admin role and then specify credentials in the configuration file like in the example below.
+Keystone should expose [Identity API v3](https://docs.openstack.org/api-ref/identity/v3/) to work with dex.
+
+
+## Configuration
+
+The following is an example of a configuration for dex config file:
```

Comment:
I don't do technical reviews for this project (yet -- still learning!), so i have a question about which file this is referring to -- in other connector examples it's referring to `examples/config-dev.yaml`, but I'm not 100% that's right either. 

The other option may be to remove this line as a little redundant (some of the other connectors don't have a blurb in their `## Configuration` section, they just go straight into the config.

```suggestion
The following is an example of an OpenStack Keystone configuration for dex:
```

Reply:
As far as I remember, we want to abstain from using `examples/config-dev.yaml`.
https://github.com/dexidp/dex/blob/master/examples/config-dev.yaml#L1

But I don't think it is a good idea to point users to another configuration file  🤔 
I think I will apply your suggestion for now.

Reply:
> As far as I remember, we want to abstain from using examples/config-dev.yaml.

I hadn't realized that--good to know.